src/main/thrift/parquet.thrift

```diff
@@ -257,7 +257,14 @@ struct Statistics {
     */
    1: optional binary max;
    2: optional binary min;
-   /** count of null value in the column */
+   /** 
+    * Count of null values in the column.
+    *
+    * Writers SHOULD always write this field even if it is zero (i.e. no null value)
+    * or the column is not nullable.
+    * Readers MUST distinguish between null_count not being present and null_count == 0.
+    * If null_count is not present, readers MUST NOT assume null_count == 0.
```

Comment:
I agree with this statement (even though there is an issue with older parquet files written with parquet-rs)

Reply:
What did parquet-rs do? Not write the null count in case it was null? If so, then readers will just not optimize out null handling in this case, which seems fine.

Reply:
@JFinis https://github.com/apache/arrow-rs/pull/6257 would support write 0 here
Before this nonthing is writen.